cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -31,12 +31,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kuberuntime "k8s.io/apimachinery/pkg/runtime"
 	clientset "k8s.io/client-go/kubernetes"
+	clientsetscheme "k8s.io/client-go/kubernetes/scheme"
```

Comment:
I wouldn't call this clientset, b/c it's int the interface layer iirc.  clientscheme is sufficient.